src/redis_bitmap.h

```diff
@@ -12,6 +12,7 @@ class Bitmap : public Database {
  public:
   Bitmap(Engine::Storage *storage, const std::string &ns): Database(storage, ns) {}
   rocksdb::Status GetBit(const Slice &user_key, uint32_t offset, bool *bit);
+  rocksdb::Status GetString(const Slice &user_key, const uint32_t &max_btos_size, std::string *value);
```

Comment:
```suggestion
  rocksdb::Status GetString(const Slice &user_key, const uint32_t max_btos_size, std::string *value);
```
for plain types, i think we don't need to use `const reference`, right?

Reply:
You are right.